task10/index.js

```diff
@@ -31,22 +31,25 @@ const getData = () => {
     });    
 };
 
-const objShow = async () => {
-    loading();
 
-    const value = await getData();
-
-    //【Question】 Can I use try-catch-finally in here?
-    // try {
-    //     getData();
-    // } catch (error) {
-    //     div.textContent = "データを読み込めませんでした";
-    //     console.error(error);
-    // }
-    // finally{
-    //     removeLoading();
-    // };
+const sequence = async () => {
+    loading();
+    try {
+        const listData = await getData();
+        return listData;
+    } catch (error) {
+        div.textContent = "データを読み込めませんでした";
+        console.error(error);
+    }
+    finally{
+        removeLoading();
+    };
+};
 
+const objShow = async () => {
+    const value = await sequence();
+    getData(value);
```

Comment:
Isn't that unnecessary?

Reply:
That's right!
I deleted it. I was being too careful.
I'll make sure to delete the code I don't need.